tests/baselines/reference/tsbuild/watchMode/programUpdates/reportErrors/when-preserveWatchOutput-is-passed-on-command-line.js

```diff
@@ -441,7 +441,7 @@ Output::
 [7m [0m [91m    ~[0m
 
 
-[[90m12:01:19 AM[0m] Found 1 error. Watching for file changes.
+[[90m12:01:22 AM[0m] Found 1 error. Watching for file changes.
```

Comment:
These timestamp changes are just because the test faked time advances a few extra ticks with these changes, right?

Reply:
Yes. Because now the tsbuildinfo is written the next time slot is used to write the output